frontend/src/app/testQueue/TestQueue.jsx

```diff
@@ -59,12 +63,23 @@ const queueQuery = gql`
 `;
 
 const TestQueue = () => {
+  const appInsights = useAppInsightsContext();
+  const trackFetchQueue = useTrackEvent(appInsights, "Fetch Queue");
   const { data, loading, error, refetch: refetchQueue } = useQuery(queueQuery, {
     fetchPolicy: "no-cache",
   });
 
+  useEffect(() => {
+    trackFetchQueue({ dummyParam: "dummyValue" }); // this works!
+  }, [trackFetchQueue]);
+
   if (error) {
-    return <p>Error in loading patients</p>;
+    // TODO: would be better to use the hooks, but we need to add an event listener
```

Comment:
There's a global graphql error hook in `frontend/src/index.js` as part of this change: https://github.com/CDCgov/prime-data-input-client/pull/203/files 

Maybe you can hook in there once it's merged in?

Reply:
hmm, lets discuss this tomorrow. We have a few different ways we are handling errors in the app, and not all of them are wired into tracking